### DISCUSSION TITLE
blog/como-programar-publicaciones-en-mastodon/

### DISCUSSION THREAD

**Discussion by giscus** [baumannzone/baumannzonedev-comments#10] [Comments]
# blog/como-programar-publicaciones-en-mastodon/

Guía completa para saber cómo programar publicaciones en Mastodon que se publiquen automáticamente. Configúralo en 3 minutos.

http://localhost:4321/blog/como-programar-publicaciones-en-mastodon/

**Comment by baumannzone** [comment]
Hola?
